### PR TITLE
Owls 76524: implement DBPing for JRF domain integ test

### PR DESCRIPTION
**Pull request by maggiehe00**
Implement DBPing for JRF domain integ test during test class prepare phase.
Internal Jenkins run: http://wls-jenkins.us.oracle.com/view/weblogic-operator/job/weblogic-kubernetes-operator-javatest/2473/consoleFull
There are 2 DBPing (searching "DB Service is Ready to accept Connection" ) happened for 2 JRF test classes.

### CODE REVIEW

**Comment by maggiehe00** [comment]
@vanajamukkara @anpanigr after you review the change, I will sync to the latest dev branch and remove the below commented out lines
   // TODO: reconsider the logic to check the db readiness
    // The jrfdomain can not find the db pod even the db pod shows ready, sleep more time
    //logger.info("waiting for the db to be visible to rcu script ...");
    //Thread.sleep(20000);

File: integration-tests/src/test/java/oracle/kubernetes/operator/utils/TestUtils.java [hunk 1]
```diff
@@ -1109,6 +1109,40 @@ public static void callShellScriptByExecToPod(
...
+   * exec into the pod and call the shell script with given arguments.
+   *
+   * @param podName pod name
+   * @param domainNS namespace
+   * @param scriptsLocInPod script location
+   * @param shScriptName script name
+   * @param args script arguments
+   * @throws Exception exception
+   */
+  public static void callShellScriptInsidePod(
```
Comment (vanajamukkara) [inline integration-tests/src/test/java/oracle/kubernetes/operator/utils/TestUtils.java]: can you see if you can use the existing method callShellScriptByExecToPod in TestUtils

Comment (maggiehe00) [reply integration-tests/src/test/java/oracle/kubernetes/operator/utils/TestUtils.java]: sure

Comment (maggiehe00) [reply integration-tests/src/test/java/oracle/kubernetes/operator/utils/TestUtils.java]: Ok, I firstly tried that method but it turned out that "-R" in below line didn't work for me
.append(" -- bash -c 'chmod +x -R ")
In my case I can only chmod for that specific file, not recursively.

Comment (vanajamukkara) [reply integration-tests/src/test/java/oracle/kubernetes/operator/utils/TestUtils.java]: I see..you can try creating a directory under /u01/oracle and see if that works with callShellScriptByExecToPod..if not you can refactor callShellScriptByExecToPod itself for your needs, for example add another argument boolean changeDirPermissions - if true use -R

```
  public static void checkDb(String[] dburl, String rcuPod) throws Exception {
  	String scriptsLocInPod = "/u01/oracle/scripts";
        TestUtils.exec("mkdir -p "+scriptsLocInPod , true);
  	TestUtils.copyFileViaCat(
        BaseTest.getProjectRoot() + "/integration-tests/src/test/resources/jrf/checkDb.sh",
        scriptsLocInPod + "/checkDb.sh",
        rcuPod,
        DEFAULT_RCU_NAMESPACE);
    TestUtils.callShellScriptInsidePod(
        rcuPod, DEFAULT_RCU_NAMESPACE, scriptsLocInPod, "checkDb.sh", dburl);
```

**Pull request closed without merging**